test/suite/test_backup14.py

```diff
@@ -273,7 +230,8 @@ def add_data_validate_backups(self):
         self.add_data(self.uri, None)
         self.take_full_backup()
         self.take_incr_backup()
-        self.compare_backups(self.uri)
+        paths = self.setup_compare_backup_paths()
+        self.compare_backups(self.uri, paths['full_backup_dir'], paths['full_backup_out'], True, paths['incr_backup_dir'], paths['incr_backup_out'])
```

Comment:
This one and all the others below are very complicated and not an improvement over the original. Is there some way to add a wrapper that just takes `paths` and pulls out those pieces in the wrapper? They're all identical as far as I see.

Reply:
yeah the redundancy, caused Tammy and I to re-purpose how compare_backups should look like, would love your insight on the new method, feels alot cleaner